boards/cats_rev1Pro/Core/Src/control/sensor_elimination.c

```diff
@@ -44,26 +43,64 @@ cats_error_e check_sensors(state_estimation_data_t *data, sensor_elimination_t *
 cats_error_e check_imus_no_faults(state_estimation_data_t *data, sensor_elimination_t *elimination) {
   /* Check bounds */
   for (int i = 0; i < 3; i++) {
-    if ((data->acceleration[i] > UPPER_BOUND_ACC) || (data->acceleration[i] < LOWER_BOUND_ACC)) {
-      elimination->faulty_imu[i] = 1;
+    if (i == ACC_100_INDEX) {
+      /* check Bound of 100G accel */
+      if ((data->acceleration[i] > UPPER_BOUND_100G_ACC) || (data->acceleration[i] < LOWER_BOUND_100G_ACC)) {
+        elimination->faulty_imu[i] = 1;
+      }
+      /* check if we are in high acceleration mode */
+      else if ((data->acceleration[i] > UPPER_BOUND_ACC) && (data->acceleration[i] < UPPER_BOUND_100G_ACC)) {
+        elimination->high_acc = true;
+      } else {
+        elimination->high_acc = false;
+      }
+    } else {
+      if ((data->acceleration[i] > UPPER_BOUND_ACC) || (data->acceleration[i] < LOWER_BOUND_ACC)) {
```

Comment:
Looks like we are eliminating the accels when we go above 32g? Is this a bug or feature?

Reply:
Well, I guess its both. But as we do not enable the accels back, I think its a bug. I will remove this constraint, as the new boolean handles this case.

Reply:
For the transition the upper bound should probably be set a little lower, we have no idea what happens close to the maximum of the sensor readout

Reply:
Lets but it to 28G?

Reply:
Maybe closer to 30-31g?

Reply:
ok